include/rdma/fabric.h

```diff
@@ -485,6 +485,7 @@ enum {
 	FI_CLASS_CONNREQ,
 	FI_CLASS_MC,
 	FI_CLASS_NIC,
+	FI_CLASS_MAX,
```

Comment:
This change needs to be dropped.  We don't want external apps trying to use this, since the intent is that the value would change between versions.

Reply:
Okay. The hook provider would need to define a FI_CLASS_MAX for ini and fini fid arrays. I'll make the changes.